test/src/test/java/org/corfudb/runtime/view/AbstractViewTest.java

```diff
@@ -469,6 +474,12 @@ public String getEndpoint(int port) {
 
     // Private
 
+    private RequestMsg getRequest(CorfuMessage.RequestPayloadMsg payloadMsg) {
+        CorfuMessage.HeaderMsg header = getHeaderMsg(0, CorfuMessage.PriorityLevel.NORMAL, 0,
+                DEFAULT_UUID, DEFAULT_UUID, true, true);
+        return getRequestMsg(header, payloadMsg);
+    }
+
```

Comment:
We need the default header for the next set of handleMessage() calls of `MANAGEMENT_BOOTSTRAP_REQUEST` and `BOOTSTRAP_SEQUENCER`.
Therefore, Can we move this to `CorfuProtocolMessage`?

We can add a new method to get default header and call `CorfuProtocolMessage.getRequestMsg()` with that header and the new `payloadMsg`.

```
public static HeaderMsg getDefaultHeaderMsg(boolean ignoreClusterId, boolean ignoreEpoch) {
    return getHeaderMsg(0, CorfuMessage.PriorityLevel.NORMAL, 0,
        DEFAULT_UUID, DEFAULT_UUID, ignoreClusterId, ignoreEpoch);
}
```
**Note:** Unfortunately, we cannot use `HeaderMsg.getDefaultInstance()` provided by Protobuf as it sets the values to empty instead of the default intended values. Its output is -
```
headerMsg.getRequestId() - 0
headerMsg.getClientId() - 
headerMsg.getClusterId() - 
headerMsg.getEpoch() - 0
headerMsg.getIgnoreEpoch() - false
headerMsg.getIgnoreClusterId() - false
headerMsg.getPriority() - PRIORITY_LEVEL_INVALID
headerMsg.getVersion() - 
```

Reply:
**Update:** After an internal discussion, we decided to add a temporary variable within the method to be reused by all 3 request messges -
```
public void bootstrapAllServers(Layout l) {
    CorfuMessage.HeaderMsg headerMsg = getHeaderMsg(0, CorfuMessage.PriorityLevel.NORMAL, 0,
            DEFAULT_UUID, DEFAULT_UUID, true, true);
```